.travis.yml

```diff
@@ -7,10 +7,10 @@ branches:
 env:
   matrix:
     # need node installs for app_manager.XpathValidatorTest
-    - TEST=python NOSE_DIVIDED_WE_RUN=05 JS_SETUP=yes
-    - TEST=python NOSE_DIVIDED_WE_RUN=6a JS_SETUP=yes
-    - TEST=python NOSE_DIVIDED_WE_RUN=bf JS_SETUP=yes
-    - TEST=python-sharded-and-javascript JS_SETUP=yes TEST_MAKE_REQUIREMENTS=yes
+#    - TEST=python NOSE_DIVIDED_WE_RUN=05 JS_SETUP=yes
+#    - TEST=python NOSE_DIVIDED_WE_RUN=6a JS_SETUP=yes
+#    - TEST=python NOSE_DIVIDED_WE_RUN=bf JS_SETUP=yes
+    - TEST=python-sharded-and-javascript JS_SETUP=yes TEST_MAKE_REQUIREMENTS=yes CITUS=yes
```

Comment:
I wanted it to be separate so that it's clear but I guess I could change `python-sharded-and-javascript` to `python-sharded-and-javascript-and-citus` or something.